features/step_definitions/sign_up_step_definitions_step.rb

```diff
@@ -0,0 +1,24 @@
+Given("I am on the landing page") do
+  visit root_path
+end
+
+When("I click on the link {string}") do |button|
+  click_on button
+end
+
+When("I fill in the {string} with {string}") do |field, content|
+  fill_in field, with: content
+end
+
+
+
+# Then("the following user exists") do |table|
+#   table.hashes.each do |name|
+#     name.create!(name)
+#   end
+#  end
+
```

Comment:
always remove comments before submitting a PR
And format the code

Reply:
fixed